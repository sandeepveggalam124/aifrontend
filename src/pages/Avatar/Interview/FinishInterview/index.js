import React from 'react';
import { useLocation } from 'react-router-dom';
import ImageComponent from '../../Landing/AvatarImage/index';
import VideoComponent from '../CaptureVideo';
import {Link} from 'react-router-dom'
import Timer from '../Timer';

const FinishInterview = () => {
  const location=useLocation()
  const avatarSrc='/assests/images/avatar.svg'
  const avatarAlt='Avatar Image'
  const containerWidth='100%'
  const containerHeight='auto'
  const interviewTime=new Date()
  interviewTime.setMinutes(interviewTime.getMinutes()+30)
  


  return (
    <div className="row custom-div text-white">
      <div className="col col-lg-1 bg-last-image"></div>
        <div className="col col-lg-10 bg-black d-flex flex-column align-item-stretch p-0 position-relative">
            {location.pathname==="/interview/finish" && <ImageComponent src={avatarSrc}  alt={avatarAlt}/>}
        {/* </div> */}
      
       {/* <div className=""> */}
       <button  className="login py-1 justify-content-center align-items-center position-absolute bottom-0 end-0 m-2" >
          <Link 
            to="/interview/end"
            className="text-decoration-none finsih-link fs-6 fw-medium">
            Finish
          </Link>
       </button>
      </div>
      <div className="col col-lg-1 bg-last-image"></div>
    </div>
  )
}

export default FinishInterview