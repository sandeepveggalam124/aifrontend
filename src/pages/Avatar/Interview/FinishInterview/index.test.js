import { MemoryRouter as Router } from "react-router-dom";
import {render} from "@testing-library/react";
import FinishInterview from "./index";


it('renders correctly',()=>{
    const {container}=render(
        <Router>
            <FinishInterview/>
        </Router>
    )
    expect(container).toMatchSnapshot()
})