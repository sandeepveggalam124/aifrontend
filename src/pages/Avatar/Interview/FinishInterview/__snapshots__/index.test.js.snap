// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="row custom-div text-white"
  >
    <div
      class="col col-lg-1 bg-last-image"
    />
    <div
      class="col col-lg-10 bg-black d-flex flex-column align-item-stretch p-0 position-relative"
    >
      <button
        class="login py-1 justify-content-center align-items-center position-absolute bottom-0 end-0 m-2"
      >
        <a
          class="text-decoration-none finsih-link fs-6 fw-medium"
          href="/interview/end"
        >
          Finish
        </a>
      </button>
    </div>
    <div
      class="col col-lg-1 bg-last-image"
    />
  </div>
</div>
`;
